### DISCUSSION TITLE
Missing "LanguageProfileId" when Syncing to Sonarr.

### DISCUSSION THREAD

**Discussion by polocabbit** [Flexget/Flexget#3787] [Q&A] [answered]
I'm building out the following task to push data from anilist over to sonarr:
```
sync-ani-to-sonarr:
	template: no_global
	anilist:
  	username: "{? anilist.username ?}"
  	status:
    	- planning
  	release_status:
    	- not_yet_released
    	- releasing
  	format:
    	- tv
	accept_all: yes
	list_add:
  	- sonarr_list:
      	base_url: "{? server.url ?}"
      	port: "{? sonarr.port ?}"
      	api_key: "{? sonarr.apikey ?}"
      	search_missing_episodes: true
      	season_folder: true
      	profile_id: "{? sonarr.quality_profile ?}"
      	only_monitored: false
      	series_type: anime
      	root_folder_path: "{? server.folders.anime ?}"
```

It pulls data as expected from anilist, but on the `list_add` comment to `sonarr_list` I get the following error:
`2023-06-04 19:33:37 CRITICAL plugin    	sync-ani-to-sonarr Sonarr returned an error. Language profile does not exist: LanguageProfileId '1'`

I validated that there is not a Language Profile in Sonarr with the Id of `1` via a GET to `/api/v3/languageprofile`, but I also do not see an option in the documentation to provide an ID for a language profile.

**Comment by gazpachoking** [comment, marked as answer]
There is indeed a language_id option to sonarr_list, but it looks like it hadn't been added to the wiki page. I added it there, and took a shot at documenting it (but I don't use sonarr myself.) Feel free to update the wiki page if needed!
